### PR TITLE
Add threading to speed lookup

### PR DESCRIPTION
**Pull request by salaheldinaz**
and PEP8 Formatting

### CODE REVIEW

File: check_online_presence.py [hunk 1]
```diff
@@ -135,30 +181,35 @@ def main():
...
         if not site['valid']:
             warn("[!] Skipping %s - Marked as not valid." % site['name'])
             continue
 
         if args.username:
-            check_site(site, args.username,
-                       if_found     = lambda url: positive("[+] User found at %s" % url),
-                       if_not_found = lambda url: neutral( "[-] User not found at %s" % url),
-                       if_neither   = lambda url: error(   "[!] Error. The check implementation is broken for %s" % url))
+            run_type = 'username'
```
Comment (mccartney) [inline check_online_presence.py]: Extracting the `message_to_print` method makes the code less readable.
And here we are dealing with potentially non-professional developers, so the readability of the code is a key factor.
Is there a better way to structure that?

Comment (salaheldinaz) [reply check_online_presence.py]: I will look into it

Comment (WebBreacher) [reply check_online_presence.py:192]: Hey @salaheldinaz any progress on this? Not pushing...just checking.

Comment (salaheldinaz) [reply check_online_presence.py:192]: > Hey @salaheldinaz any progress on this? Not pushing...just checking.
I missed this notification.
I will continue work on it  👍

**Pull request closed without merging**